borg/testsuite/convert.py

```diff
@@ -0,0 +1,163 @@
+import os
```

Comment:
the module you test is named "converter", so should this rather be "testsuite.converter"?